ansible/roles/base/tasks/main.yml

```diff
@@ -10,6 +10,26 @@
 - name: Set DHCP config
   template: src=etc/dhcp/dhclient.conf.j2 dest=/etc/dhcp/dhclient.conf
 
+# Node
+- name: Register NodeJS
+  shell: 'curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -'
+  changed_when: false
+
+# Yarn
+- name: Install Yarn APT key
```

Comment:
This would be better in a "yarn" role. Instead of downloading a key for an apt repo it is safer to store it locally and load it. See the node role referenced above for an example.